app/helpers/version_helper.rb

```diff
@@ -79,19 +83,22 @@ def show_past_versions(obj, args = {})
       link = "#{:VERSION.t} #{ver.version}"
       link += " " + ver.format_name.t if ver.respond_to?(:format_name)
       if ver.version != obj.version
-        if @merge_source_id
-          link = link_with_query(link, controller: obj.show_controller,
-                                       action: "show_past_#{type}", id: obj.id,
+        link = if @merge_source_id
```

Comment:
This is a Codeclimate issue. See [Rails/HelperInstanceVariable](https://docs.rubocop.org/rubocop-rails/2.7/cops_rails.html#railshelperinstancevariable)
I approved it because:
- it was pre-existing and 
- the fix would entail many changes which are irrelevant to this PR and likely would create a cascade of new Codeclimate issues.